scripts/qa_crowbarsetup.sh

```diff
@@ -2437,6 +2482,11 @@ function deploy_single_proposal()
 
     # create proposal
     case "$proposal" in
+        nfs_client)
+            if [[ -n "$clusternodesservices" ]]; then
```

Comment:
There are lots of other examples where we cleaned this up already.
But never mind, this just reminds me, that I wanted to do a big cleanup commit for this long ago :)